README.md

```diff
@@ -43,8 +43,31 @@ The Azure AI Content Safety API uses severity levels from 0-6:
 
 This repository is licensed under the [Mattermost Source Available License](LICENSE) license.
 
+## Frequently Asked Questions
+
+### How does content moderation work?
+
+When a user posts a message, it appears immediately in the channel. The plugin then analyzes the content in the background using Azure AI Content Safety APIs. If harmful content is detected, the post is automatically deleted and notifications are sent to inform users of the removal.
+
+### Will I still receive notifications for harmful content?
+
+Currently, yes. Push notifications may be sent for posts that contain harmful content before the moderation process completes. This is because notifications are typically sent immediately when posts are created, while content analysis happens asynchronously. We are working to improve this behavior (see roadmap).
+
+### Can I exclude certain users from moderation?
+
+Yes, you can specify user IDs in the "Excluded Users" configuration setting. All other users will have their content moderated automatically.
```

Comment:
Does this support user groups?

Reply:
Yes it does, but the README doesn't mention it - forgot to add that. I'll make that change :+1:

Reply:
> No - but we do support excluding by channel. The README doesn't mention it - forgot to add that. This is now included below :+1: 

Ok, let's create a ticket to support groups in future.